desktop/windows/install.md

```diff
@@ -149,8 +151,9 @@ The install command accepts the following flags:
 - `--backend=<backend name>`: selects the backend to use for Docker Desktop, `hyper-v` or `wsl-2` (default)
 
 If your admin account is different to your user account, you must add the user to the **docker-users** group:
-```
-net localgroup docker-users <user> /add
+
+```console
+C:\> net localgroup docker-users <user> /add
```

Comment:
Here again, if we prefix with a command prompt it seems to say it’s not for e.g. PowerShell.
But actually the same command runs fine in PowerShell or even WinGW, …